cypress/e2e/cross-chain/cross_chain_x_to_p.cy.ts

```diff
@@ -0,0 +1,199 @@
+import {
+    BN,
+    bnToAvaxX,
+    bnToBigAvaxX,
+    bnToBigAvaxP,
+    Big,
+} from '@c4tplatform/camino-wallet-sdk'
```

Comment:
iirc, the import should come from a `dist` folder, so `'@c4tplatform/camino-wallet-sdk/dist'`. Same goes for every other test file. Am I right here @peak3d ?